emitter/src/contract.rs

```diff
@@ -57,8 +57,10 @@ impl Emitter for EmitterContract {
 
         storage::set_backstop(&e, &backstop);
         storage::set_blend_id(&e, &blnd_token_id);
+        storage::set_last_fork(&e, e.ledger().sequence() - 777600); // We set the block 45 days in the past to allow for an immediate initial drop
+        storage::set_drop_status(&e, false);
         // TODO: Determine if setting the last distro time here is appropriate, since it means tokens immediately start being distributed
-        storage::set_last_distro_time(&e, &(e.ledger().timestamp() - 7 * 24 * 60 * 60));
+        storage::set_last_distro_time(&e, &0);
```

Comment:
was this a mistake?

We probably do want an issue to track when we want distribution to begin, as this has been a placeholder for awhile.